src/main/java/com/authlete/common/dto/ClientExtension.java

```diff
@@ -380,4 +384,68 @@ public ClientExtension setTokenExchangePermitted(boolean permitted)
 
         return this;
     }
+
```

Comment:
Could you add another empty line here? (We normally have two empty lines between methods)